pinot-orc/src/main/java/org/apache/pinot/orc/data/readers/ORCRecordReader.java

```diff
@@ -186,6 +192,20 @@ private Object getBaseObject(WritableComparable w) {
     return obj;
   }
 
+  private List<Object> translateList(OrcList<? extends WritableComparable> l) {
+    if (l == null || l.size() < 1) {
```

Comment:
If an isEmpty() method is exposed, it might be better to use that instead of size() < 1.